modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenModel.java

```diff
@@ -41,6 +41,7 @@
 
     public Set<String> imports = new TreeSet<String>();
     public boolean hasVars, emptyVars, hasMoreModels, hasEnums, isEnum, hasRequired, hasOptional, isArrayModel, hasChildren;
+    public boolean isInteger, isFloat;
```

Comment:
This flag names a bit misleading for me (same names exists in CodegenProperty), may be isFloatEnum/isIntegerEnum?

Reply:
Yes, this flag also exists in CodegenProperty. And it has the same name as it has the same meaning.

Imaging this definition:
```
definitions:
  IntegerValue:
    type: integer
    format: int32
  Object:
    type: object
    properties:
      IntegerValue:
        type: integer
        format: int32
```

The outer ```IntegerValue``` is an integer, so the flag ```isInteger == true```.
The inner ```IntegerValue``` is an integer, so the flag ```isInteger == true```.

Why should I call the flag differently?

Reply:
@wing328 plz advice.

Reply:
Previously we encounter the use case with array of array of enum. To make it easier to check whether the inner type is an integer, we use `isInteger`. If we need to ensure it's not an enum, we can `{{^isEnum}} .. {{/isEnum}}`

I agree it may be a bit misleading and it's tradeoff to make it easier to handle 2D or 3D array of enum